### ISSUE TITLE
Missing support for dimensions?

### ISSUE THREAD

**Issue by alexandervlpl** [michael-brade/LaTeX.js#59]
Many thanks for your amazing work on this! I was about to celebrate this excellent find when I ran into a limitation:

```
ERROR: Parsing failure:

line 5 (column 1): \\usepackage{lipsum}\n\\newdimen\\fontdim\n\\n
-----------------------------------------^
expected \begin{document}
```
Looks like dimensions are not supported? :cry: Is this in the pipeline?

**Comment by michael-brade** [comment]
Yeah, `\newskip` and `\newdimen` are lowlevel TeX. I didn't implement much (if anything) of TeX yet, mostly LaTeX. I do want to add more of TeX's syntax, but first I want to get what is already there close to bugfree. It's a slow process because I don't have enough time to do all that I want to do...
